op-service/sources/l1_beacon_client.go

```diff
@@ -205,7 +205,7 @@ func (cl *L1BeaconClient) GetTimeToSlotFn(ctx context.Context) (TimeToSlotFn, er
 }
 
 func (cl *L1BeaconClient) fetchSidecars(ctx context.Context, slot uint64, hashes []eth.IndexedBlobHash) (eth.APIGetBlobSidecarsResponse, error) {
-	var errs []error
```

Comment:
Not all cases result in errors here so this may allocate extra, not clear which is better

Reply:
If you revert this diff then its an obvious merge

Reply:
thank you for feedback, i updated